tests/src/main/java/org/hibernate/beanvalidation/tck/tests/xmlconfiguration/ClockProviderSpecifiedInValidationXmlNoDefaultConstructorTest.java

```diff
@@ -24,7 +22,12 @@
  * @author Guillaume Smet
  */
 @SpecVersion(spec = "beanvalidation", version = "2.0.0")
-public class ClockProviderSpecifiedInValidationXmlNoDefaultConstructorTest extends AbstractTCKTest {
+public class ClockProviderSpecifiedInValidationXmlNoDefaultConstructorTest extends AbstractBootstrapFailureTCKTest {
+
+	@Override
+	protected Class<? extends Exception> acceptedDeploymentExceptionType() {
+		return ValidationException.class;
```

Comment:
Does it actually work in the case of the OP? It seems the `ValidationException` was wrapped there? Or are you trying to find the root exception?

Reply:
I suppose you have your answer now :).

I browse through the causes to try to find a valid exception. It might not be perfect but I hope it's going to work for the OP.

Reply:
Ok. Once we got the list of tests, it'd be nice to deploy this as a snapshot and ask them to give it a spin.

Reply:
I think I will ask them to build my branch and test if it's OK for the commented tests.